src/main/scala/mesosphere/marathon/storage/StorageConf.scala

```diff
@@ -34,6 +34,12 @@ trait StorageConf extends ZookeeperConf with BackupConf {
     default = Some(32)
   )
 
+  lazy val groupVersionsCacheSize = opt[Int](
```

Comment:
What value did we use before? I cannot find it in this diff.

Reply:
Wait, why is it a new parameter at all? GCActor does use `maxVersions` variable defined above for exactly these purpose, doesn't it?

Reply:
there are 2 parameters: GC actor's and repo cache size. They're independent